test/runtests.jl

```diff
@@ -198,4 +215,94 @@ facts("Test integration of ODE with complex dependent variables") do
 
 end
 
+facts("Test jet transport: simple pendulum") do
+
+    function pendulum!(t, x, dx) #the simple pendulum ODE
+        dx[1] = x[2]
+        dx[2] = -sin(x[1])
+    end
+
+    const varorder = 2 #the order of the variational expansion
```

Comment:
Yes, thanks! I agree that the equations of motion should return `nothing` in the multi-dim case!